agent/roles/manifests/templates/agent-config_yaml.j2

```diff
@@ -14,8 +14,11 @@ rendezvousIP: {{ ips[0] }}
 {% else %}
 rendezvousIP: {{ ipsv6[0] }}
 {% endif %}
-{% if boot_mode == "PXE" %}
-bootArtifactsBaseURL: {{ pxe_server_url }}
+{% if (boot_mode == "PXE") or (agent_minimal_iso == "true" and mirror_images) %}
```

Comment:
Could you explain the relationship between agent_minimal_iso and mirror_images? Does having AGENT_MINIMAL_ISO set to true require a user to also set MIRROR_IMAGES to true?

Reply:
AGENT_MINIMAL_ISO doesn't imply that MIRROR_IMAGES is needed as the rootfs will be downloaded from mirror.openshift.com in a connected setup. However, if MIRROR_IMAGES is set, then the `bootArtifactsBaseURL` must also be set since the rootfs in that case must be retrieved from the local boot server.

I will add a comment describing this.